pages/common/git-clean.md

```diff
@@ -0,0 +1,23 @@
+# git clean
+
+> Remove untracked files from the working tree.
+
+- Delete files that are not tracked by git:
+
+`git clean`
+
+- Interactively delete files that are not tracked by git:
+
+`git clean -i`
+
+- Shows what files would be deleted without actually deleting them:
+
+`git clean --dry-run`
+
+- Force delete files that are not tracked by git:
```

Comment:
To use "delete" as the verb, it should be "force-delete" or "forcefully delete".